sources/packages/api/src/utilities/esdc-utils.ts

```diff
@@ -12,6 +12,18 @@ export function getMaritalStatusCode(
   return maritalStatus === RelationshipStatus.Single ? "S" : "O";
 }
 
+/**
+ * Gets the Part-Time marital status code conversion for the MSFAA request file
+ */
+export function getPTMaritalStatusCode(
```

Comment:
just for my information. is `MaritalStatusCode` different for PT and FT?

Reply:
Yes